src/depth-to-rgb-calibration.h

```diff
@@ -40,11 +41,14 @@ namespace librealsense
             std::vector< algo::depth_to_rgb_calibration::yuy_t > const & last_yuy_data,
             algo::depth_to_rgb_calibration::algo_calibration_info const & cal_info,
             algo::depth_to_rgb_calibration::algo_calibration_registers const & cal_regs,
+            rs2_intrinsics yuy_intrinsics,
```

Comment:
Why?

Reply:
we need to correct the thermal by the raw intrinsics